workflows/eager.nf

```diff
@@ -202,28 +202,28 @@ workflow EAGER {
             ch_bams_from_input = ch_samplesheet_bams.join(SAMTOOLS_INDEX_BAM_INPUT.out.bai)
         }
```

Comment:
Is indexing of the original input BAMs actually needed anymore?
I don't think it is required for samtools merge.